### DISCUSSION TITLE
select boot target based on partial mac

### DISCUSSION THREAD

**Discussion by mdidomenico4** [ipxe/ipxe#231] [Q&A]
i'm new to ipxe scripts, so forgive me if this is naive.  i have two different server mfg on my network, is there anyway to auto select a boot target based on the first three divisions of the mac address?  

pseudo code

if mac == 00:00:02
  boot targetb
else
  boot targeta

**Comment by mcb30** [comment]
Not purely within the scripting language, no.  Your easiest solution is probably to send the MAC address to a web server and let the server choose what to boot.  For example `chain http://webserver/boot?mac=${mac}`
